client/web/src/features/series/seriesSaga.ts

```diff
@@ -12,6 +12,13 @@ export function* fetchSeries() {
   })
 }
 
+export function* fetchOneSeries(action: FetchOneSeriesAction) {
+  const { id } = action
+  yield put({ type: actionTypes.UNSET_CURRENT_SERIES })
```

Comment:
Why unset current series ?

Reply:
if there is one error on fetch (not found for example), I don't want to show the previous selected series